README.template.md

```diff
@@ -59,39 +59,42 @@ There are a number of `docker-compose.yml` files provided by this repository:
 
 ## Override
 
-This repository ignores `docker-compose.override.yml` which will be included in
-any `docker compose` commands you invoke by default.
+This git repository does not track `docker-compose.override.yml` which will 
+be included by default in all `docker compose` commands you invoke.
 
 Two platform dependent templates that allow for access to the hosts `SSH agent`
 are provided for `development` environments.
 
-Simply copy the appropriate `docker-compose.PLATFORM.yml` file into
+Simply copy the appropriate* `docker-compose.PLATFORM.yml` file into
 `docker-compose.override.yml`, on your development machine.
 
 Any additional changes that are for your local / development environment can
-then be added to `docker-compose.override.yml`.
+then be added to `docker-compose.override.yml`. Because they're not tracked...
 
 ## Building
 
-You can build your locally using `docker compose`
+You can build your development environment locally using `docker compose`
 
 ```bash
 docker compose --profile dev build
 ```
+This has the effect of...
 
-## Pulling
+## Pulling Docker Images
 
-The `docker compose` file provided require that you first pull the islandora
+The `docker compose` file provided requires that you first pull the islandora
```

Comment:
"first" implies... before building (previous step)? Before the next step? If that's the case why not just state this as the next step?

Reply:
Before anything can happen within a container you must download that image to build the container with. It sounds like this is the intent.

Reply:
It is implied that this requirement is a quirk of our particular `docker-compose.yml` file.

Reply:
It's also after the 'build' step which confuses me.